unit_tests/stencil_composition/sid/test_sid_shift_sid_origin.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2019, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <gtest/gtest.h>
+
+#include <gridtools/common/defs.hpp>
+#include <gridtools/common/hymap.hpp>
+#include <gridtools/common/tuple_util.hpp>
+#include <gridtools/stencil_composition/dim.hpp>
+#include <gridtools/stencil_composition/sid/shifted_sid.hpp>
+#include <gridtools/stencil_composition/sid/synthetic.hpp>
+
+namespace gridtools {
+    namespace {
+        using sid::property;
+        using namespace literals;
+        namespace tu = tuple_util;
+
+        TEST(sid_helpers, shifted) {
+            double data[3][5][7];
+
+            auto src = sid::synthetic()
```

Comment:
now you can use `data` as well if you want

Reply:
I add both cases as they are using different makers